packages/recommender/src/__tests__/integration/recommender.integration.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
```

Comment:
These test files are just moved files and not new additions, right? Are there new things you added besides refactoring imports to make it work with the new project structure?

Reply:
This one is new.

Reply:
Nice 👍